recipe/meta.yaml

```diff
@@ -24,9 +24,6 @@ outputs:
     script: build_base.sh  # [not win]
     build:
       skip: true  # [py<38]
-      ignore_run_exports:
-        - tk  # [linux]
-        - zlib
     requirements:
       build:
         - pkg-config  # [not win]
```

Comment:
any specific reason why we don't make use of pkg-config on Windows too?

Reply:
I tested and the package was built successfully on Windows with pkg-config unspecified for unix-only. I am not entirely sure whether it used it or not, though.